dagshub/upload/wrapper.py

```diff
@@ -341,6 +341,27 @@ def full_name(self):
         return f"{self.owner}/{self.name}"
 
 
+
+    def _get_last_commit(self):
+        """
+        The _get_last_commit function returns the last commit sha for a given branch.
+        It is used to check if there are any new commits in the repo since we last ran our dag.
```

Comment:
Great catch! Didn't notice that this function isn't actually in the object it gets called at 🙃